.lighthouserc.json

```diff
@@ -2,7 +2,8 @@
   "ci": {
     "collect": {
       "numberOfRuns": 1,
-      "startServerCommand": "python -m scripts.start --no_browser --prod_env",
+      "startServerCommand": "python -m scripts.start",
```

Comment:
Why is this modified?

Reply:
Oh yes, I modified it just for testing. It's much faster. Before I merge I will change it back

Reply:
I think it's best that you don't push it as a part of the changes. You might forget to update it before the merge.

Reply:
As @jameesjohn said, remove this please.

Reply:
Done